src/v0/destinations/am/transform.js

```diff
@@ -335,14 +335,65 @@ const getDefaultResponseData = (message, rawPayload, evType, groupInfo) => {
   const groups = groupInfo && cloneDeep(groupInfo);
   return { groups, rawPayload };
 };
+
```

Comment:
move this new function to am/utils.js & have few basic unit tests

Reply:
added